Week1/homework.md

```diff
@@ -0,0 +1,52 @@
+2. Assignment
+Using Slack application on desktop and mobile:
+
+Describe at least 2 shared testing activities for desktop and mobile versions
+1-responsivness which include testing  on different screen sizes. 
+2-GUI (graphical user interface) testing. 
+3-Testing API endpoints.
+
+Given RealWorld example application on web (React/Redux) and on mobile (NativeScript (Angular))
+Describe shared testing activities for desktop and mobile
+1-Check what is the maximum size set for the input.
+2-Check what kind of charechters does it allow. 
+3-Check if the database is vulnerable for sql injection.
+4-Check what happens when you enter different combination of inputs, example 
+a-Wrong username, wrong password. 
+b-Wrong username, correct password. 
+c-Correct username, correct password. 
+d-Special charechters in both fields. 
+5-Check if the validation is done on server side and also on client. 
+6-Check if the username can be copy pasted. 
+7-Chack if there is a minimal charechters should be entered to login. 
+8-Graphical user interface testing.
+9-Recovery testing
+10-Responsivness. 
+11-Check menus Home,sign in , sign up. 
+12-Testing the app on different devices.
+13-Security testing.
+14-Bug fixes
+15-Stress testing.
+16-Check if any cookies are stored during login.
+17-Speed test.
+18-Check buttons or Links to different pages.
+19-Crowd Testing.
+
+
+Describe unique testing activities per platform
+1-web application  
+-Testing if the connection between web browsers and the server is running smoothly.
```

Comment:
This could be in the sharing test activity for mobile and desktop because the defect here will have an effect no matter what device you are using